03-Make/soup/check_dirs.sh

```diff
@@ -0,0 +1,9 @@
+! #bin/bash
```

Comment:
What is the meaning of ! #/bin/bash?
Do You mean #!/bin/bash ?

Reply:
Yes )